ivy/functional/ivy/elementwise.py

```diff
@@ -5210,13 +5210,13 @@ def tan(
 
     >>> x = ivy.array([0.5, -0.7, 2.4])
     >>> y = ivy.zeros(3)
-    >>> ivy.tan(x, out=y)
+    >>> ivy.tan(x,out=y)
     >>> print(y)
     ivy.array([0.546, -0.842, -0.916])
 
     >>> x = ivy.array([[1.1, 2.2, 3.3],\
                         [-4.4, -5.5, -6.6]])
-    >>> ivy.tan(x, out=x)
+    >>> ivy.tan(x,out=x)
```

Comment:
Even though this does not cause a formatting error as it is in the examples, but in actural code, you should add a space between the two arguments right after the comma. So I guess you should not make these changes to the examples.